drivers/SmartThings/samsung-audio/src/handlers.lua

```diff
@@ -134,4 +135,11 @@ function CapabilityHandlers.handle_set_volume(driver, device, cmd)
   end
 end
 
+function CapabilityHandlers.handle_audio_notification(driver, device, cmd)
+  local ip = device:get_field("ip")
+  log.info(string.format("RETRY for SPECIFIER ----> Received audio notification with uri %s", cmd.args.uri))
```

Comment:
Done. Thanks for sharing. Log was just for experimentation purpose to confirm the End to End working.
Since its working fine, I removed the log now.